src/main/groovy/life/qbic/samplestatus/reporter/services/NcctLocationService.groovy

```diff
@@ -23,8 +25,17 @@ class NcctLocationService implements LocationService {
     @Autowired
     private SampleTrackingService sampleTrackingService
 
+    @Autowired
+    private UserService userService
+
     @Override
-    Optional<Location> getCurrentLocation() {
+    Location getCurrentLocation() {
```

Comment:
Why was this removed? The Optional facilitates idiomaic usage of safe value access and avoid NPEs.